src/main/java/com/alkemy/ong/service/CategoryService.java

```diff
@@ -26,10 +30,26 @@ public Category createCategory(Category category) {
     }
 
     @Transactional
-    public List<Category> getAll() {
-        List<CategoryModel> categoryModelList = categoryRepository.findAll();
-        return categoryModelList.stream().map(CategoryMapper::mapModelToDomain)
-                .collect(Collectors.toList());
+    public PageDTO<Category> findAll(int page) throws CategoryNotFoundException {
+        if (page < 0) {
+            throw new CategoryNotFoundException(String.format("Category with page: %s not found", page));
+        }
```

Comment:
Este control debería ir en la capa controller, y si mandan un valor no númerico o menor a 0, debería devolver un bad request, no un not found 👌🏼  pero por el momento con chequear lo de menor a 0 alcanza.